modules/assim.sequential/R/sda.enkf.R

```diff
@@ -225,6 +229,9 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
   
   for (i in seq_len(nens)) {
     
+    # is this gonna break other model runs? inputs is usually just the met path which is all they need anyway?
+    settings$run$inputs <- inputs[[i]]
```

Comment:
This could break things, but hopefully it's a very temporary change since much of this will change with the refactor